.travis.yml

```diff
@@ -0,0 +1,49 @@
+# Travis CI (MIT License) configuration file
+# @link https://travis-ci.org/
+
+# Use new container based environment
+sudo: false
+
+# Declare project language.
+# @link http://about.travis-ci.org/docs/user/languages/php/
+language: php
+
+# Declare versions of PHP to use. Use one decimal max.
+php:
+    # Aliased to a recent high PHP version
+    - 5.6
+    # Might as well test HHVM too
+    - hhvm
+    # Lowest currently possible in travis
+    - 5.2
```

Comment:
Doesn't Travis only support 5.3 min now?

What about 5.3, 5.4, 5.5? Occasionally something breaks there, and currently we would only know that it passed the low PHP level, but failed at 5.6.